fastmri/data/mri_data.py

```diff
@@ -294,12 +294,12 @@ def __init__(
                 example for example in self.examples if example[0].stem in sampled_vols
             ]
 
-        if num_cols:
-            self.examples = [
-                ex
-                for ex in self.examples
-                if ex[2]["encoding_size"][1] in num_cols  # type: ignore
-            ]
```

Comment:
@Timsey do you know where this change came from?

Reply:
Hmm, this is probably an erroneous delete. We should put this back.